crypto/bn/bn_prime.c

```diff
@@ -250,6 +250,12 @@ int ossl_bn_check_prime(const BIGNUM *w, int checks, BN_CTX *ctx,
     return bn_is_prime_int(w, checks, ctx, do_trial_division, cb);
 }
 
+int ossl_bn_check_prime_fast(const BIGNUM *w, int checks, BN_CTX *ctx,
+                             int do_trial_division, BN_GENCB *cb)
```

Comment:
We probably also do not need the do_trial_division argument here if we keep bn_is_prime_int as is.

Reply:
You mean it would always be 1?

Reply:
Yes